### ISSUE TITLE
Active item not visible with dark theme

### ISSUE THREAD

**Issue by lzybkr** [octobox/octobox#1628]
Are you experiencing this issue in octobox.io or your own instance?
 - [x] octobox.io
 - [ ] My instance

🐞 **Problem**

I'm trying to use keyboard navigation with the dark theme and I can't see any indication of which item is selected. The bar is visible with the light theme - maybe a little subtle but when it moves it's obvious.

🎯 **Goal**

Better support keyboard navigation with the dark theme.

**Comment by andrew** [comment]
PR with an improvement for this: https://github.com/octobox/octobox/pull/1805

![Screenshot 2019-05-06 at 15 51 52](https://user-images.githubusercontent.com/1060/57233962-96ea3180-7017-11e9-9bb3-c5506d959f90.png)
